fibonacci/fibonacci.py

```diff
@@ -0,0 +1,18 @@
+num=int(input("Enter the number of elements of the series:"))
```

Comment:
use proper spacing b/w operands and operators.

Reply:
ok..
modified it

Reply:
ok